mobx_codegen/lib/src/template/store.dart

```diff
@@ -56,6 +60,30 @@ abstract class StoreTemplate {
 
   $actions''';
 
+    if (generateToString) {
+      for(var i = 0; i < observables.templates.length; i++){
+        final ObservableTemplate current =  observables.templates[i];
+        toStringList.add('${current.name}: \${${current.name}.toString()}');
+      }
+
+      for(var i = 0; i < computeds.templates.length; i++){
+        final ObservableTemplate current =  computeds.templates[i];
+        toStringList.add('${current.name}: \${${current.name}.toString()}');
+      }
```

Comment:
I think about creating a Template class with name. ObservableTemplate and ComputedTemplate would extend this class and I could do something like
```dart
for(var i = 0; i < T.templates.length; i++){
        final T current =  computeds.templates[i];
        toStringList.add('${current.name}: \${${current.name}.toString()}');
      }
```

Reply:
I don't think you should extend `ObservableTemplate` or `ComputedTemplate`. Instead add, a **`toFieldValueString`** as a getter and use that as part of the `toString` inside the `StoreTemplate`